bga_database/static/js/helloworld.js

```diff
@@ -1,4 +1,20 @@
-export function helloWorld () {
-	console.log("Hello, world!");
-	return;
+export function getYearData () {
```

Comment:
What do you think about defining and exporting the async function in this module, and then binding it to the change event in `index.js`? I think it makes it a little easier to trace what's going on that way, but happy for your thoughts!